intellij-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -90,6 +90,17 @@
                      level="WEAK WARNING"
     />
 
+    <!-- "Schema in .graphql file" inspection -->
+    <!--suppress PluginXmlCapitalization -->
+    <localInspection language="GraphQL"
+                     implementationClass="com.apollographql.ijplugin.inspection.ApolloSchemaInGraphqlFileInspection"
+                     groupPathKey="inspection.group.graphql"
+                     groupKey="inspection.group.graphql.apolloKotlin"
+                     key="inspection.schemaInGraphqlFile.displayName"
+                     enabledByDefault="true"
+                     level="ERROR"
```

Comment:
Maybe just `WARNING` ?

Reply:
But this breaks the build right?

Reply:
IIUC, The inspection is for all `.graphql` files? I.e not only the ones that are compiled by Apollo Kotlin, right? Someone could have a `.graphql` file in the project somewhere for reference or documentation purpose. In that case `WARNING` feels a bit safer.